### PR TITLE
feat(client): make tables mobile friendly, updated carousel, small UI changes 

### PR DESCRIPTION
**Pull request by sujvn**
#

## Changes this PR introduces

- Tables are mobile friendly
- Changed sidebar menu color to blue, added logo to navbar
- update carousel

## How to test this PR

- test tables in mobile resolution

## Notes for reviewers

-

## Checklist

- [x] I renamed the title of my pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-rn75rt Ensure tables are mobile friendly](https://app.clickup.com/t/rn75rt)

File: client/src/components/NavBar.jsx [hunk 1]
```diff
@@ -51,8 +51,21 @@ const NavBar = () => {
...
+        paddingLeft={userStore.loggedIn ? '3rem' : '7rem'}
+        fontSize={{ base: '14px', sm: '36px' }}
+        lineHeight={{ base: '40px', sm: '1.5' }}
+        textAlign="center"
+        display="flex"
+        flexDirection="column"
+      >
+        <Image
+          src="/images/headerLogo.png"
+          alt="No results illustration"
```
Comment (gordonpn) [inline client/src/components/NavBar.jsx]: `No results illustration`??

**Review by gordonpn — COMMENTED** [review]
schedulings, sales and quality management tables the pagination moves with the horizontal scrolling

File: client/src/components/AuditTrail/AuditTrailTable.jsx [hunk 2]
```diff
@@ -40,26 +40,28 @@ const AuditTrailTable = () => {
...
-            searchData(data.data)
-              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
-              .map((log, index) => (
-                <Fragment key={index}>
-                  <AuditTrailTableRow log={log} />
-                </Fragment>
-              ))}
-        </Tbody>
-      </Table>
+      <Box overflowX="auto">
```
Comment (gordonpn) [inline client/src/components/AuditTrail/AuditTrailTable.jsx:43]:
```suggestion
      <Box overflowX="scroll">
```

**Comment by KarinKazarian** [comment]
Some tables are looking like this
image

**Comment by KarinKazarian** [comment]
should we align the elements in the header?
image

**Review by gordonpn — DISMISSED** [review]
Good work the brown kid Sujan!

**Review by Mackbellemore — DISMISSED** [review]
very aesthetic

**Review by gordonpn — APPROVED** [review]
Approved

**Review by Mackbellemore — APPROVED** [review]

**Pull request merged**